src/material/form-field/form-field.md

```diff
@@ -129,7 +129,7 @@ the prefix. Similarly, adding `matSuffix` will designate it as the suffix.
 In addition to the form field controls that Angular Material provides, it is possible to create
 custom form field controls that work with `<mat-form-field>` in the same way. For additional
 information on this see the guide on
-[Creating Custom mat-form-field Controls](https://material.angular.io/guide/creating-a-custom-form-field-control).
+[Creating Custom mat-form-field Controls](https://v7.material.angular.io/guide/creating-a-custom-form-field-control).
```

Comment:
I think that this will link people on the 8.0 site to the 7.0 version of the guide as well. A better solution might be to change the link to `/guide/creating-a-custom-form-field-control`.

Reply:
Updated!